packages/jsii-python-runtime/tests/test_compliance.py

```diff
@@ -899,10 +886,9 @@ def test_eraseUnsetDataValues():
     assert not EraseUndefinedHashValues.does_key_exist(opts, "option2")
 
 
-@xfail_callbacks
 def test_objectIdDoesNotGetReallocatedWhenTheConstructorPassesThisOut():
     class PartiallyInitializedThisConsumerImpl(PartiallyInitializedThisConsumer):
-        def consume_partially_initialized_this(self):
+        def consume_partially_initialized_this(self, obj, dt, en):
```

Comment:
Would be nice if you can also assert that `dt` and `en` have the correct values (like the Java implementation does).